### PR TITLE
[Docs] Japanese translation of docs/feature_encoders.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

Add a Japanese translation of docs/feature_encoders.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw よかったらレビューお願いします。

File: docs/ja/feature_encoders.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#define ENCODERS_PAD_B { encoder1b, encoder2b }
+```
+
+エンコーダの時計回りの方向が間違っている場合は、A & B パッド定義を交換することができます。define を使って逆にすることもできます:
+
+```c
+#define ENCODER_DIRECTION_FLIP
+```
+
+さらに、解像度を同じファイルで指定することができます (デフォルト & お勧めは4):
```
Comment (shelaf) [inline docs/ja/feature_encoders.md:34]:
```suggestion
さらに、解像度を同じファイルで指定することができます (デフォルトかつお勧めは4):
```

File: docs/ja/feature_encoders.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        } else {
+            tap_code(KC_UP);
+        }
+    }
+}
+```
+
+## ハードウェア
+
+エンコーダの A と B のラインは MCU に直接配線し、C/common ラインはグランドに配線する必要があります。
```
Comment (shelaf) [inline docs/ja/feature_encoders.md:81]:
```suggestion
エンコーダの A と B の線は MCU に直接配線し、C/common 線はグランドに配線する必要があります。
```

File: docs/ja/feature_encoders.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+各 PAD_A/B 変数は配列を定義するため、複数のエンコーダを定義することができます。例えば:
+
+```c
+#define ENCODERS_PAD_A { encoder1a, encoder2a }
+#define ENCODERS_PAD_B { encoder1b, encoder2b }
+```
+
+エンコーダの時計回りの方向が間違っている場合は、A & B パッド定義を交換することができます。define を使って逆にすることもできます:
```
Comment (shelaf) [inline docs/ja/feature_encoders.md:28]:
```suggestion
エンコーダの時計回りの方向が間違っている場合は、A と B のパッド定義を交換することができます。define を使って逆にすることもできます:
```

File: docs/ja/feature_encoders.md (hunk 1 above)
Comment (umi-umi) [reply docs/ja/feature_encoders.md]: ご指摘ありがとうございます。

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by noroadsleft** [comment]
どうもありがとうございました！

**Pull request merged**